packages/react/src/components/Tab/Tab.js

```diff
@@ -161,12 +161,12 @@ export default class Tab extends React.Component {
         tabIndex={-1}
         className={classes}
         onClick={evt => {
-          handleTabClick(index, label, evt);
+          handleTabClick(index, evt, disabled);
```

Comment:
```suggestion
          if (disabled) {
            return;
          }
```

rather than passing the `disabled` prop around, we can just ignore anything if it's `true` right?